### DISCUSSION TITLE
How can I configure to see full log?

### DISCUSSION THREAD

**Discussion by cooliyev** [opcodesio/log-viewer#327] [Q&A] [answered]
Hello, let's imagine I have a log more than 128 kb.
Log viewer says that `The contents of this log have been cut short to the first 128.00 KB. The full size of this log entry is 463.06 KB`
How can I configure to see full log?

**Comment by arukompas** [comment, marked as answer]
hey @cooliyev 

so far it's kind of a hidden/undocumented feature, but there's a solution. Add this to your AppServiceProvider:

```php
LogViewer::setMaxLogSize($bytes);
```

**Reply by cooliyev** [reply]
Oh, thank you!
